### ISSUE TITLE
How can i use tag with tabs

### ISSUE THREAD

**Issue by zhangyyue** [ptournet/obsidian-html-tabs#1]
I found out i can’t use tag inside <content>

**Comment by ptournet** [comment]
> I found out i can’t use tag inside

Can you explain what you are trying to achieve ? (best with an actual sample of what doesn't work)

**Comment by ptournet** [comment]
Yes, that is one of the features that need to be implemented. But, as of version 1.1.0, I have found a way to tackle links and tags should work in a not too different way, so I should be able to iron that soon.

**Comment by zhangyyue** [comment]
I know，thanks a lot
